pkg/controller/workload/job/job_controller.go

```diff
@@ -83,6 +83,7 @@ func (r *JobReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+//+kubebuilder:rbac:groups="",resources=events,verbs=create;watch;update
```

Comment:
can we also add this to the core controllers as well just in case this controller gets refactored out?

Reply:
well we can do a diff thing, we can add this to kustomize , example
```yaml
patchesJSON6902:
- target:
    kind: ClusterRole
    name: manager-role
  patch: |-
    - op: add
      path: /rules/0
      value:
        apiGroups:
        - ""
        resources:
        - events
        verbs:
        - create
        - update
  ```

Reply:
this way we don't worry about refactors in the go code, and kustomize keeps us in place

Reply:
wouldn't jus adding this line to the other controllers be simpler? can we add it to the [main.go](https://github.com/kubernetes-sigs/kueue/blob/main/main.go) file for example or does it have to be in the one of the controllers files?

Reply:
did to the controllers as initially requested